### PR TITLE
Query parameters Tests

### PR DESCRIPTION
**Pull request by Fewwy**
Wrote 2 tests that check the additional query parameters for the Recs list page 
Added fixtures to the tests for single Cluster page
Created new tests to test the query parameters
Improved previous tests we had for the single Cluster page
image

### CODE REVIEW

**Comment by Fewwy** [comment]
@quarckster Ptal

**Review by quarckster — APPROVED** [review]
LGTM. Thanks!

File: src/Components/Cluster/Cluster.spec.ct.js [hunk 1]
```diff
@@ -55,7 +77,10 @@ describe('cluster page', () => {
     cy.get(CLUSTER_HEADER).should('have.length', 1);
     // renders table component
     cy.get(RULES_TABLE).should('have.length', 1);
+    // test how many rows were rendered
+    cy.getAllRows().should('have.length', 10);
```
Comment (ikerreyes) [inline src/Components/Cluster/Cluster.spec.ct.js:81]: You have the data, so it is better to count the number of expected rows from there directly. This way, the data can be expanded without affecting the test

File: src/Components/Cluster/Cluster.spec.ct.js [hunk 2]
```diff
@@ -110,4 +136,46 @@ describe('cluster page', () => {
     cy.get(RULES_TABLE).should('have.length', 0);
     cy.get('.pf-c-empty-state').should('have.length', 1);
   });
+
+  it('adds additional filters passed by the query parameters №1', () => {
+    mount(
+      <MemoryRouter initialEntries={['?total_risk=1&text=foo+bar&category=2']}>
```
Comment (ikerreyes) [inline src/Components/Cluster/Cluster.spec.ct.js:142]: Just curiosity, why the parameter is a list?

**Review by ikerreyes — COMMENTED** [review]
Overall looks good to me

**Pull request merged**